src/main/java/Duke.java

```diff
@@ -19,26 +25,76 @@ public static void main(String[] args) {
 //                    "\n____________________________________________________________\n";
 //            System.out.println(output);
 //            input = sc.nextLine();
-//        } Level-1
-        String[] myList;
-        myList = new String[100];
+//        }
+//        Level-1
+//        String[] myList;
+//        myList = new String[100];
+//        Scanner sc = new Scanner(System.in);
+//        String input = sc.nextLine();
+//        int numberOfInputs = 0;
+//        while(!input.equals("list")){
+//            myList[numberOfInputs] = input;
+//            numberOfInputs++;
+//            String output = "____________________________________________________________\n " +
+//                    "Added: " +
+//                    input +
+//                    "\n____________________________________________________________\n";
+//            System.out.println(output);
+//            input = sc.nextLine();
+//        }
+//        for(int i = 0; i < numberOfInputs; i++){
+//            System.out.print(i + 1);
+//            System.out.println(". " + myList[i]);
+//        }
+//        Level-2
+
+
+        Task[] myList;
+        myList = new Task[1000];
         Scanner sc = new Scanner(System.in);
```

Comment:
All names should be written in English. "sc" is not indicative of what the variable does. Perhaps you could give it a name that tells readers this sc is a scanner that gets user input?